frontend/src/bundles/friends/pages/friends/friends.tsx

```diff
@@ -0,0 +1,143 @@
+                await dispatch(
+                    friendsActions.addFollowing({ followingId: id }),
+                );
+            };
+            void addFollowing(id);
+        },
+        [dispatch],
+    );
+
+    const handleRemoveFollowing = useCallback(
+        (id: number): void => {
+            const removeFollowing = async (id: number): Promise<void> => {
+                await dispatch(
+                    friendsActions.removeFollowing({ followingId: id }),
+                );
+            };
+            void removeFollowing(id);
+        },
+        [dispatch],
+    );
+
+    useEffect(() => {
+        const loadUsers = async (): Promise<void> => {
+            activeTab === TabsFollowers.FIND_FOLLOWINGS &&
+                (await dispatch(friendsActions.getNotFollowed()));
+            activeTab === TabsFollowers.MY_FOLLOWINGS &&
+                (await dispatch(friendsActions.getFollowings()));
+        };
+        void loadUsers();
+    }, [dispatch, activeTab]);
+
+    return (
+        <section className="relative flex flex-col gap-5 whitespace-normal">
+            <Tabs
+                tabs={tabs}
+                handleTabClick={handleTabClick}
+                activeTab={activeTab}
+            />
+            {loading === DataStatus.FULFILLED ? (
+                <div
+                    className={`flex flex-wrap items-start gap-5 ${selectedCard?.id ? 'w-[calc(100%-354px)]' : 'w-full'}`}
+                >
+                    {activeTab === TabsFollowers.FIND_FOLLOWINGS && (
+                        <TabContent
+                            users={users}
+                            isFollowed={false}
+                            selectedCardId={selectedCard?.id}
+                            selectCard={handleSelectCard}
+                            onToggleFollow={handleAddFollowing}
+                            noUsersText={'No user found to follow.'}
+                        />
+                    )}
+
+                    {activeTab === TabsFollowers.MY_FOLLOWINGS && (
+                        <TabContent
+                            users={users}
+                            isFollowed={true}
+                            selectedCardId={selectedCard?.id}
+                            selectCard={handleSelectCard}
+                            onToggleFollow={handleRemoveFollowing}
+                            noUsersText={'You do not follow anyone yet.'}
+                        />
+                    )}
+
+                    {activeTab === TabsFollowers.MY_FOLLOWERS && (
+                        <div
+                            className={
+                                'text-primary flex h-[70vh] w-full items-center justify-center text-xl'
+                            }
+                        >
+                            No one is following you yet.
+                        </div>
+                    )}
+                </div>
+            ) : (
+                <Loader />
```

Comment:
isOverflow for centered loader

Reply:
And also add `relative` to the parent of `<Loader />`